src/box/lua/config/instance_config.lua

```diff
@@ -2300,6 +2300,19 @@ return schema.new('instance_config', schema.record({
         -- are configured in the `config.etcd` or the
         -- `config.storage` section.
         stateboard = schema.record({
+            -- Stateboard enabled/disabled status.
+            --
+            -- Beware: disabling stateboard leads to the
+            -- following consequences:
+            -- * Failover commands aren't supported.
+            -- * Failover state monitoring isn't provided.
+            -- * Multiple coordinators over the same replicaset
+            --   behave in an unpredictable way. Two RW instances
+            --   are possible.
```

Comment:
Same here: for me the key point behind this option is that now the coordinator doesn't start by default without a configured remote config/state storage and that one who needs it should use the option. We tell about consequences of disabling, but I think the main idea should be explained first.

Reply:
Dito. I've changed the comment here. Now it's started from the main idea that by default one of the remote configuration sources should be set.